src/main/java/com/iota/iri/service/ValidationException.java

```diff
@@ -1,6 +1,6 @@
 package com.iota.iri.service;
 
-public class ValidationException extends Exception {
+public class ValidationException extends RuntimeException {
```

Comment:
This is required so that the functional command routing doesnt need try/catch -> throw another error